### DISCUSSION TITLE
[@types/ramda]: Latest version includes removed functions

### DISCUSSION THREAD

**Discussion by asbjorn-brevio** [DefinitelyTyped/DefinitelyTyped#64001] [Issues with a @types package]
Ramda 0.28 removed a bunch of functions (see https://github.com/ramda/ramda/issues/3218):
- composeK
- composeP
- contains
- merge
- pipeK
- pipeP

These functions seem to still be exported from `@types/ramda@0.28.21`, which results in runtime errors without any type error

[a comment by an automated account is not shown]

**Comment by Nicholaiii** [comment]
A suitable PR is more than welcome ❤️

**Reply by asbjorn-brevio** [reply]
I don't know about suitable, but there's now a PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64050
